strings/horspool.cpp

```diff
@@ -0,0 +1,81 @@
+//
+// Implementation of Horspool's algorithm is an algorithm for finding substrings in strings
+//
+// The All ▲lgorithms Project
+//
+// https://allalgorithms.com/strings
+// https://github.com/allalgorithms/cpp
+//
+// Contributed by: Hemant Singh
+// Github: @Robcross
+#include<iostream>
+#include<conio.h>
+#include<string.h>
+#include<stdio.h>
+using namespace std;
+
+class horspool
+{
+	private: char p[40],t[40];
+	int m,n,s[256];
+	public: void shift();int match();
+	void read();
+};
+
+void horspool:: read ()
+{
+	cout<<"enter text string\n";
+	gets(t);

```

Comment:
- Remove `gets()` from `<conio.h>`

* It can be replaced with `cout` from `<iostream>`